src/layered_config_tree/main.py

```diff
@@ -346,33 +346,80 @@ def to_dict(self) -> dict[str, Any]:
                 result[name] = child.to_dict()
         return result
 
-    def get(self, key: str, default_value: Any = None) -> Any:
-        """Return the LayeredConfigTree or value at the key in the outermost layer of the config tree.
+    def get(self, keys: str | list[str], default_value: Any = None) -> Any:
+        """Return the value at the key or key path in the outermost layer.
 
         Parameters
         ----------
-        key
-            The string to look for in the outermost layer of the config tree.
+        keys
+            The string or ordered list of strings to look for in the tree
+            starting from the outermost layer.
         default_value
-            The value to return if key is not found
+            The value to return if and only if the *final* key in the key path
+            does not exist.
+
+        Notes
+        -----
+        The ``default_value`` will only be used if *final* key in the key path
+        does not exist *but the rest of the key path does*.
+
+        Returns
+        -------
+        The value at the key or nested keys. If the key path exists except for the
+        final value, the ``default_value`` is returned.
         """
-        return self[key] if key in self._children else default_value
+        if not isinstance(keys, (str, list)):
```

Comment:
Is this necessary? We don't usually do things like this, but since this is a library method, it does make sense to give the user a better error message.